apis/api-directory/cloud-cost-api.md

```diff
@@ -0,0 +1,55 @@
+# Cloud Cost API
+
+{% swagger method="get" path="/cloudcost/query" baseUrl="http://app.kubecost.com/query" summary="Cloud Cost API (Querying)" %}
+{% swagger-description %}
+The Cloud Cost API is the preferred way to query for costs and resources related to cloud provider services.
+{% endswagger-description %}
+
+{% swagger-parameter in="path" name="window" required="true" type="string" %}
+Duration of time over which to query. Accepts multiple formats including units of time, relative time units, and unix timestamps. See this section on [using the `window` parameter](/apis/api-directory/api-directory.md#using-the-window-parameter) for more information.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="format" type="string" required="false" %}
+File type when exporting query. Currently only supports `json`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="aggregate" type="string" required="false" %}
+Used to consolidate cost model data. Supported values are `invoiceEntityID`, `accountID`, `provider`, `providerID`, `category`, and `service`, as well as `label:<name>`. Passing an empty value for this parameter or none at all returns data by an individual asset. Supports multi-aggregation (aggregation of multiple categories) in a comma separated list, such as `aggregate=account,project`.

```

Comment:
These are **all** the supported agg options for cloud. No further action required here.